src/lib.rs

```diff
@@ -159,6 +194,11 @@ impl Client {
         self.fetch::<Account>(&format!("/accounts/{}", address))
     }
 
+    /// Get wallet information for the richest accounts
+    pub fn get_accounts_richest(&self, limit: Option<u64>) -> error::Result<Vec<Account>> {
```

Comment:
a u64 for limit is perhaps a bit overlay optimistic at how large pages can be.. u16 or u32 at best